conjur/client.py

```diff
@@ -30,84 +32,33 @@ class Client():
 
     _api = None
     _login_id = None
-    _api_key = None
-
-    LOGGING_FORMAT = '%(asctime)s %(levelname)s: %(message)s'
-
 
     # The method signature is long but we want to explicitly control
-    # what paramteres are allowed
-    #pylint: disable=too-many-arguments,too-many-locals
+    # what parameters are allowed
+    # pylint: disable=too-many-arguments,too-many-locals
     def __init__(self,
-                 account=None,
-                 api_key=None,
-                 ca_bundle=None,
-                 debug=False,
                  http_debug=False,
-                 login_id=None,
-                 password=None,
-                 ssl_verify=True,
-                 url=None):
```

Comment:
See my other comment but this is really not a good direction to take architecturally

Reply:
answered above (: